src/components/text-field/react/TextField.tsx

```diff
@@ -22,6 +22,9 @@ enum TextFieldType {
  * Defines the props of the component.
  */
 interface ITextFieldProps extends IGenericProps {
+    /** A Chip Group to be rendered before the main text input */
+    chips?: HTMLElement | ReactNode;
```

Comment:
Stop me if I'm wrong but, if the goal is to have a TextField with multiple values, why not pass an array to the `value` prop ?
If `value` is an array, then I render a ChipGroup into the TextField.

Reply:
Because they are separate things that will be rendered. On the one hand, we have the list of chips, which are already selected values. On the other hand, we have the value itself, which is the text to be displayed on the Text Field. Imagine the Tagz component, that has a list of chips on the left and a text on the right.

Reply:
![Screenshot 2019-10-25 at 12 04 32](https://user-images.githubusercontent.com/13719066/67562760-a4a78680-f71f-11e9-9215-46cd61182644.png)

Reply:
And the `ReactNode` type in typescript is a union type that actually includes `Array<ReactNode>` (yes it's a recursive reference ^^)

Reply:
Ok :ok_hand: